src/components/signing_official_table/SigningOfficialTable.js

```diff
@@ -198,6 +198,10 @@ const generateLcaText = (text) => {
   </ReactMarkdown>;
 };
 
+const onlyResearchersWithoutCardFilter = (researcher) => {
+  return isNil(researcher.libraryCards);
```

Comment:
I can imagine an edge case where a researcher belongs to this institution, doesn't have a library card from this institution, but does have another library card (since admins can technically give researchers library cards from any institution). In that case, they'll appear in the table but not the dropdown in the modal. Not sure how big of an issue this is though

Reply:
Great catch @shaemarks ... and you're right. There will be cases where a user will have library cards from other institutions.

Reply:
changed the filter here and also the filter in the table itself to check that the SO's institution id the same as the card's institution id